src/Data/Profunctor/Static.hs

```diff
@@ -0,0 +1,41 @@
+{-# LANGUAGE StaticPointers #-}
+
+module Data.Profunctor.Static (StaticProfunctor(..)) where
+
+import Control.Distributed.Closure
+import Data.Typeable (Typeable)
+
+-- | Instances of 'StaticProfunctor' should satisfy the following laws:
+--
+-- @
+-- 'staticDimap' (static id) (static id) = static id
+-- 'staticLmap' (static id) = static id
+-- 'staticRmap' (static id) = static id
+-- 'staticDimap' f g = staticLmap f . staticRmap g
```

Comment:
Kmett claims that, by parametricity, these laws imply the laws for composition. I haven't worked out how it works yet, so I don't know whether that holds for closures as well (though I assume it would).